docs/docs/backup-and-restore.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Backup and Restore
+---
+
+# Backup and Restore
+
+Cassandra on Mesos supports backup and restore operation. The API for those operation is exposed through REST.
```

Comment:
`The API for those operation is exposed through REST` -> `These operations are exposed through a REST API.`